packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -265,7 +267,7 @@ export const prepareVaultDirector = (
           return harden({});
         },
         getGovernedApiNames() {
-          return harden({});
+          return harden([]);
```

Comment:
!

Reply:
amended 9c4e487d30b6fbc3d07edd1932e47671c4e80aa8 after review to prevent drifts in the future